cypress/e2e/inventory/search/browse-call-numbers-all.cy.js

```diff
@@ -0,0 +1,134 @@
+import { DevTeams, Permissions, TestTypes } from '../../../support/dictionary';
+import Users from '../../../support/fragments/users/users';
+import TopMenu from '../../../support/fragments/topMenu';
+import getRandomPostfix from '../../../support/utils/stringTools';
+import InventoryInstances from '../../../support/fragments/inventory/inventoryInstances';
+import InventorySearchAndFilter from '../../../support/fragments/inventory/inventorySearchAndFilter';
+import Location from '../../../support/fragments/settings/tenant/locations/newLocation';
+import ServicePoints from '../../../support/fragments/settings/tenant/servicePoints/servicePoints';
+import { BROWSE_CALL_NUMBER_OPTIONS } from '../../../support/constants';
+import InventoryInstance from '../../../support/fragments/inventory/inventoryInstance';
+import ItemRecordNew from '../../../support/fragments/inventory/item/itemRecordNew';
+import BrowseCallNumber from '../../../support/fragments/inventory/search/browseCallNumber';
+
+describe('Call Number Browse', () => {
```

Comment:
Please, use two describe. One for 'Inventory', second one for Call Number Browse'

Reply:
done